packages/marketplace/contracts/transfer-manager/TransferManager.sol

```diff
@@ -14,11 +15,15 @@ import {LibPart} from "../lib-part/LibPart.sol";
 /// @notice responsible for transferring all Assets
 /// @dev this manager supports different types of fees
 /// @dev also it supports different beneficiaries
-abstract contract TransferManager is ERC165Upgradeable, ITransferManager {
+abstract contract TransferManager is ERC165Upgradeable, AccessControlUpgradeable, ITransferManager {
```

Comment:
I think it's better to only have access control in one place and on the top contract (Exchange).
What we could do here:
- add a virtual internal function `_skipFees`
- extend it in Exchange, and protect it with the role

Reply:
example 
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/proxy/utils/UUPSUpgradeable.sol#L128C6-L128C76